src/pages/video-test.tsx

```diff
@@ -100,100 +151,102 @@ const VideoTest: React.FC<any> = ({videoResource}) => {
   const onProgress = () => {}
 
   return (
-    <div>
+    <div className="-mx-5 -mt-5">
```

Comment:
I don't understand the `Page` component below. Why not add `PlayerProvider` here above the div instead?

Then you could just export this component and don't need the extra lines for Yet Another Wrapper.

I might be missing something.

Reply:
I had to wrap the whole thing for getCueNotes to work.

Reply:
threw me too lol!